src/instructlab/model/backends/backends.py

```diff
@@ -113,10 +113,11 @@ def is_model_gguf(model_path: pathlib.Path) -> bool:
         return first_four_bytes_int == GGUF_MAGIC
 
 
-def determine_backend(model_path: pathlib.Path) -> str:
+def determine_backend(logger: logging.Logger, model_path: pathlib.Path) -> str:
```

Comment:
Please use a modul-level logger instead of passing loggers around. We got mostly rid of this bad practice a short while ago.

Reply:
Ah right, forgot about that, thanks for pointing it out.

Reply:
PTAL https://github.com/instructlab/instructlab/pull/1604